qa/pull-tester/rpc-tests.py

```diff
@@ -153,48 +161,35 @@
 ]
 
 def runtests():
+    test_list = []
+    if '-extended' in opts:
+        test_list = testScripts + testScriptsExt
+    elif len(opts) == 0 or (len(opts) == 1 and "-win" in opts):
+        test_list = testScripts
+    else:
+        for t in testScripts + testScriptsExt:
+            if t in opts or re.sub(".py$", "", t) in opts:
+                test_list.append(t)
+
+    if print_help:
+        # Help should be the same for all scripts, so just
+        # call the first and exit
```

Comment:
This is true, but the previous behavior was to only show the help text of the first script and then `exit(0)`. If someone wants to change the behavior, I'd propose a separate pull. I am happy to change the help text in this pull, though.